Source/Core/DolphinQt/Settings/AudioPane.cpp

```diff
@@ -160,10 +160,22 @@ void AudioPane::CreateWidgets()
 
   dsp_box->setSizePolicy(QSizePolicy::Expanding, QSizePolicy::Fixed);
 
+  auto* misc_box = new QGroupBox(tr("Miscellaneous Settings"));
+  auto* misc_layout = new QGridLayout;
+  misc_box->setLayout(misc_layout);
+
+  m_speed_up_mute_enable = new QCheckBox(tr("Mute When Disabling Speed Limit."));
+  m_speed_up_mute_enable->setToolTip(
+      tr("When overriding the emulation speed limit (default hotkey: Tab), mute the audio"));
```

Comment:
```suggestion
  m_speed_up_mute_enable = new QCheckBox(tr("Mute When Disabling Speed Limit"));
  m_speed_up_mute_enable->setToolTip(
      tr("When overriding the emulation speed limit (default hotkey: Tab), mutes the audio."));
```

Reply:
Alternatively: `Mutes the audio when overriding the emulation speed limit (default hotkey: Tab).`